### PR TITLE
[Hotfix] Typo in folder name

### PR DESCRIPTION
**Pull request by jcotela**
unnitest -> unittest, no other changes.

### CODE REVIEW

File: kratos/mpi/tests/sources/test_mpi_data_communicator.cpp [hunk 1]
```diff
@@ -2438,5 +2438,10 @@ KRATOS_DISTRIBUTED_TEST_CASE_IN_SUITE(MPIDataCommunicatorErrorBroadcasting, Krat
     );
 }
 
+KRATOS_DISTRIBUTED_TEST_CASE_IN_SUITE(MPITestPartialFailure, KratosMPICoreFastSuite)
+{
+    KRATOS_CHECK_EQUAL(DataCommunicator::GetDefault().Rank(), 1);
```
Comment (philbucher) [inline kratos/mpi/tests/sources/test_mpi_data_communicator.cpp]: I don't think this should be here

**Review by loumalouomega — DISMISSED** [review]
OK

File: kratos/mpi/tests/sources/test_mpi_data_communicator.cpp (hunk 1 above)
Comment (jcotela) [reply kratos/mpi/tests/sources/test_mpi_data_communicator.cpp]: You are right, will revert.

**Review by philbucher — APPROVED** [review]
:+1:

**Pull request merged**